### PR TITLE
Numerical mappings filter

### PR DESCRIPTION
**Pull request by chrilehner**
see #233 and Caleydo/ordino#22

### CODE REVIEW

File: src/mappingeditor.ts [hunk 1]
```diff
@@ -250,11 +272,82 @@ export default class MappingEditor {
...
+        .on('change', () => {
+          const element = <HTMLInputElement>(<MouseEvent>d3event).target;
+          const type = (<any>element.dataset).type;
+
+          let position = 0;
+          switch(type) {
+            case 'normalized':
+              d.n = element.value;
+              position = normal2pixel(parseFloat(element.value));
+              select(this).select('line').attr('x1', position);
```
Comment (sgratzl) [inline src/mappingeditor.ts]: what is `this` in this context?

**Comment by sgratzl** [comment]
2017-04-18 13_56_07-lineup alpha

a couple of styling questions: 
 * atm the place where you can specify the filter values (min filter, ...) are not visually linked to their interactive versions
 * why is the order in the overlay reversed to the order that is shown in the mapping editor (normalized, raw)?
 * why don't both input fields don't have the same with in the overlay?
 * in the overlay they are called `normalized scores` whereas now in the mapping editor: `LineUp Scores`
 * when I click on a blue button and the overlay appears and then drag the blue button to the right, the line/blue button goes behind the overlay such that I can't see what is going on

filtering: 
 * there was a dedicated notation of -Inf and +Inf to say that the filter should not be applied. Now it seems that it will be set to input domain bounds. However, when I change the input domains bounds the filters are not updated

**Comment by mstreit** [comment]
also round the values in a reasonable way (e.g., two decimal places)

**Comment by chrilehner** [comment]
The styling issues should be resolved now in Chrome

The filtering should now work as before. But please test and if it behaves differently, please let me know. But I tested it again and I think it should be correct now

**Comment by mstreit** [comment]
Is something preventing us from merging this soon?

**Pull request merged**